README.md

```diff
@@ -20,7 +20,7 @@ A fluid multi-table layout. Each table groups subjects together under a shared p
 
 1. Copy this lens into your `refocus-ldk/Lenses` directory.
 
-        cp -r ../refocus-lens-multitable/MultiTable Lenses/MultiTable/
```

Comment:
the old command created a MultiTable folder inside the refocus-lens-multitable/MultiTable

Reply:
I don't understand why removing this instruction line.

Reply:
Depends on what directory you're in when you run this, right?